EIPS/eip-4626.md

```diff
@@ -572,7 +572,7 @@ For production implementations of Vaults which do not use ERC-4626, wrapper adap
 
 ## Reference Implementations
 
-See [Solmate ERC4626](https://github.com/Rari-Capital/solmate/blob/main/src/mixins/ERC4626.sol):
+See [Solmate ERC4626](https://github.com/transmissions11/solmate/blob/main/src/mixins/ERC4626.sol):
```

Comment:
EIPs shouldn't have external links.  Please inline this code or add it to `../assets/eip-4626/` and include a relative link here.  Note: Must be CC0 licensed.